dashboard/src/t5gweb/utils.py

```diff
@@ -186,7 +186,11 @@ def set_cfg():
         if os.environ.get("jira_labels")
         else []
     )
-
+    cfg["sla_settings"] = (
+        json.loads(os.environ.get("sla_settings"))
+        if os.environ.get("sla_settings")
+        else None
```

Comment:
so on a similar note, maybe this should be:
~~~
if os.environ.get("sla_settings"):
    cfg["sla_settings"] = json.loads(os.environ.get("sla_settings"))
~~~
That way if its not set via env, it will still use that default

Reply:
ack, changed too.